### PR TITLE
feat: view my lists link

### PR DESCRIPTION
**Pull request by Heitors09**
## Describe your changes
Added a button that takes me to the page of my lists, which only appears for the authenticated user and already renders in their lists tab when clicked. To achieve this, I created a client component that opens the tabs of the page, handles the fetches within it, and passes their results as props to my component and subsequently to the components that the shdcn tabs render.

## Issue ticket number and link
#172
PLO-52

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it's an essential feature, I've tested it thoroughly.
- [ ] Do we need to implement analytics?
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

a button who leads the user from the all lists page to the own lists page

Preview:
image

### CODE REVIEW

File: apps/web/public/dictionaries/ja-JP.json [hunk 1]
```diff
@@ -289,7 +289,8 @@
   "list_page": {
     "list_not_found": "リストが見つかりませんでした。",
     "see_your_lists_or_create_new": "リストを見るか、ここをクリックして新しいリストを作成する",
-    "here": "ここ"
+    "here": "ここ",
```
Comment (eulixir) [inline apps/web/public/dictionaries/ja-JP.json:292]:
```suggestion
    "here": "ここに",
```

File: apps/web/public/dictionaries/en-US.json [hunk 2]
```diff
@@ -289,7 +289,8 @@
   "list_page": {
     "list_not_found": "List not found.",
     "see_your_lists_or_create_new": "See your lists or create new clicking",
-    "here": "here"
+    "here": "here",
+    "list_view_link": "View lists"
```
[the remark this answers is not in the document]
Comment (eulixir) [reply apps/web/public/dictionaries/en-US.json:293]: I agree with @lui7henrique , just "here" has no connection, we need to use a phrase to make sense like "click here".

**Comment by Heitors09** [comment]
Changes made:

- Styles aligned with those provided in the review.
- Use of context in the profile-tabs component and removal of language from dictionary as props.
- Changed from object to variable in the TypeScript dictionary, "view all".
- Conflits with the main

File: apps/web/public/dictionaries/en-US.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Heitors09) [reply apps/web/public/dictionaries/en-US.json]: changes in the line "here" as being shown because of the ","

**Pull request merged**